core-bundle/src/Resources/contao/templates/forms/form_captcha.html5

```diff
@@ -15,34 +15,14 @@
 
   <input type="text" name="<?= $this->name ?>" id="ctrl_<?= $this->id ?>" class="captcha mandatory<?php if ($this->class): ?> <?= $this->class ?><?php endif; ?>" value="" aria-describedby="captcha_text_<?= $this->id ?>"<?= $this->getAttributes() ?>>
```

Comment:
I had to replace this entire template just to add `form-control` bootstrap classes and now with this change I will have to update again. Maybe we could introduce a block just for the part that is shown to the user in case the captcha has to be entered manually? I think that styling might be adjusted in projects but overriding the entire template will make it hard to keep track of such updates. Maybe we could also fix that in this PR?

Reply:
I would prefer to fix this in a separate PR.

Reply:
I tried it, but it’s not possible because of `Nested blocks are not allowed in child templates` as this template itself extends `form_row.html5`

> I had to replace this entire template just to add `form-control` bootstrap

Did you try the following?

```php
<?php 
$this->extend('form_captcha');
$this->class = $this->class . ' form-control';
```

Reply:
Yes, I'll do this in the future :)